packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -148,11 +148,14 @@ export function buildComponentsList(
   metadataType: string
 ): string[] {
   try {
-    const fileData = JSON.parse(fs.readFileSync(componentsPath, 'utf8'));
     const metaComponents = [];
-    for (const component of fileData) {
-      if (!isNullOrUndefined(component.fullName)) {
-        metaComponents.push(component.fullName);
+    const fileData = fs.readFileSync(componentsPath, 'utf8');
+    if (fileData !== 'undefined') {
```

Comment:
This is for covering when the file gets created bu the content is undefined instead of a JSON structure, right ?

Reply:
Yup, exactly